src/typography/display2/TextDisplay2.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { ReactNode } from 'react'
+
+import Text, { TextDisplayType, TextTagType } from 'text'
+
+export interface TextBodyProps {
+  readonly className?: Classcat.Class
+  readonly children: string | ReactNode
+  readonly isInverted?: boolean
+  readonly role?: string
+  readonly ariaLabel?: string
+}
+
+const TextDiplay2 = ({ children, className, isInverted = false, ...props }: TextBodyProps) => {
+  const classes = isInverted ? `kirk-text--inverse ${className}` : className
+  return (
+    <Text display={TextDisplayType.DISPLAY2} tag={TextTagType.SPAN} className={classes} {...props}>
```

Comment:
Why do we use Text here a simple <span> with the good styles should be enough ?
I think Ideally we should get rid of Text.

Reply:
To do not introduce a hard break change, we could update that "later" on each component

Reply:
You don't need to delete the `Text` component, however not using it here will save you time when we'll remove it